src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -3115,6 +3115,9 @@
   <data name="Overflow_UInt64" xml:space="preserve">
     <value>Value was either too large or too small for a UInt64.</value>
   </data>
+  <data name="PlatformNotSupported_CodeBase" xml:space="preserve">
+    <value>CodeBase is not supported on assemblies loaded from memory.</value>
```

Comment:
I think this is a bit confusing. For assemblies loaded from byte-arrays, `CodeBase` doesn't throw, but returns the caller's location. So, it is better to say assemblies loaded from single-file bundle.